src/ResourceManager/RecoveryServices.SiteRecovery/Commands.RecoveryServices.SiteRecovery/packages.config

```diff
@@ -2,5 +2,8 @@
 <packages>
   <package id="AutoMapper" version="6.0.2" targetFramework="net452" />
   <package id="Microsoft.Azure.Management.RecoveryServices" version="4.2.0-preview" targetFramework="net45" />
-  <package id="Microsoft.Azure.Management.RecoveryServices.SiteRecovery" version="1.0.3-preview" targetFramework="net452" />
-</packages>
+  <package id="Microsoft.Azure.Management.RecoveryServices.SiteRecovery" version="1.2.0-preview" targetFramework="net452" />
+  <package id="Microsoft.Rest.ClientRuntime" version="2.3.11" targetFramework="net452" />
```

Comment:
The ClientRuntime packages are handed by the Common.Dependencies.targets and Profile's packages.config respectively. These will not be used.